.gcloudignore

```diff
@@ -0,0 +1,29 @@
+# This file specifies files that are *not* uploaded to Google Cloud
+# using gcloud. It follows the same syntax as .gitignore, with the addition of
+# "#!include" directives (which insert the entries of the given .gitignore-style
+# file at that point).
+
+^(.*/)?#.*#$
```

Comment:
Is there a reason these have to be so complicated? The first two seem to be to prevent temporary files. I'm not sure what the google cloud process creates, but if they're not autogenerated I'm not sure if we need these.

The remaining ones can be simplified to: (python cache/object files are already ignored lower down)
*~
.*
.*/**